### PR TITLE
OCPBUGS#34724: Removed incorrect vSphere link and replaced step

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.14+

Issue:
[OCPBUGS-34724](https://issues.redhat.com/browse/OCPBUGS-34724)

Link to docs preview:
https://83887--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating-hardware-on-nodes-running-on-vsphere.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jinyunma** [comment]
@sgaoshang could you help to review this? thanks.

File: modules/update-vsphere-virtual-hardware-on-template.adoc [hunk 1]
```diff
@@ -21,6 +21,11 @@ in the VMware documentation prior to the next step.
 Once converted from a template, do not power on the virtual machine.
 ====
 
-. Update the VM in the vSphere client. Follow link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-60768C2F-72E1-42E0-8A17-CA76849F2950.html[Upgrade the Compatibility of a Virtual Machine Manually] in the VMware documentation for more information.
-. Convert the VM in the vSphere client from a VM to template. Follow link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.hostclient.doc/GUID-846238E4-A1E3-4A28-B230-33BDD1D57454.html[Convert a Virtual Machine to a Template in the vSphere Client] in the VMware documentation for more information.
+. Update the virtual machine (VM) in the vSphere client. Follow link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-60768C2F-72E1-42E0-8A17-CA76849F2950.html[Upgrade the Compatibility of a Virtual Machine Manually] in the VMware documentation for more information.
```
Comment (dfitzmau) [inline modules/update-vsphere-virtual-hardware-on-template.adoc:24]:
```suggestion
. Update the virtual machine (VM) in the {vmw-full} client. Complete the steps outlined in link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-60768C2F-72E1-42E0-8A17-CA76849F2950.html[Upgrade the Compatibility of a Virtual Machine Manually] ({vmw-full} documentation).
```

File: modules/update-vsphere-virtual-hardware-on-template.adoc [hunk 2]
```diff
@@ -21,6 +21,11 @@ in the VMware documentation prior to the next step.
...
+. Convert the VM in the vSphere client from a VM to a template. Right-click on the VM and select **Template -> Convert to Template**.
```
Comment (dfitzmau) [inline modules/update-vsphere-virtual-hardware-on-template.adoc:25]:
```suggestion
. Convert the VM in the {vmw-short} client to a template by right-clicking on the VM and then selecting **Template -> Convert to Template**. 
```

File: modules/update-vsphere-virtual-hardware-on-template.adoc [hunk 3]
```diff
@@ -21,6 +21,11 @@ in the VMware documentation prior to the next step.
...
 ====
 
-. Update the VM in the vSphere client. Follow link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-60768C2F-72E1-42E0-8A17-CA76849F2950.html[Upgrade the Compatibility of a Virtual Machine Manually] in the VMware documentation for more information.
-. Convert the VM in the vSphere client from a VM to template. Follow link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.hostclient.doc/GUID-846238E4-A1E3-4A28-B230-33BDD1D57454.html[Convert a Virtual Machine to a Template in the vSphere Client] in the VMware documentation for more information.
+. Update the virtual machine (VM) in the vSphere client. Follow link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-60768C2F-72E1-42E0-8A17-CA76849F2950.html[Upgrade the Compatibility of a Virtual Machine Manually] in the VMware documentation for more information.
+. Convert the VM in the vSphere client from a VM to a template. Right-click on the VM and select **Template -> Convert to Template**. 
++
+[IMPORTANT]
+====
+The steps for converting a VM to a template may change in future vSphere versions.
```
Comment (dfitzmau) [inline modules/update-vsphere-virtual-hardware-on-template.adoc:29]:
```suggestion
The steps for converting a VM to a template might change in future {vmw-short} documentation versions.
```

**Review by dfitzmau — APPROVED** [review]

**Pull request merged**